tox.ini

```diff
@@ -7,10 +7,11 @@ deps =
     coverage
 setenv = PYTHONDEVMODE = 1
 commands =
-    coverage run --concurrency=multiprocessing -m pytest --doctest-modules --pyargs xopen tests
```

Comment:
I also just noticed that we have coverage configuration in both `setup.cfg` and `tox.ini` (my fault). We can get rid of `setup.cfg` actually because the only other thing in it is the "universal wheel" configuration, and we don’t need that since we’re not compatible with Python 2 anymore.

Reply:
I removed setup.cfg.